### ISSUE TITLE
[Research] Notification Prefs - Refine deliverables for notification settings research

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#24166]
**Background**

In sprint 46, we started the [research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/notifications/notification-preferences/discovery-and-research/notifications-research-plan.md) and [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/notifications/notification-preferences/discovery-and-research/notifications-conversation-guide.md) for our notification settings research.  Now we need to update the prototype + documentation to prepare for a study in early/mid June.

**Tasks**

- [x] Update prototype (add contact info screens, ensure alignment between groups + notification items to tasks + MVP definition, remove "note:..." copy)
- [x] Update plan + convo accordingly
- [x] Review w/ team
- [x] Ready for mid-point review

**Comment by andaleliz** [comment]
@Samara-Strauss I'm re-thinking the layout of the notification groups now that I see them with the long labels we have in our MVP.  Here's how what I originally proposed translates to the current 3 column pattern we have in other parts of profile:
<img src="https://images.zenhubusercontent.com/5d9def68b6de6d00018d50f2/5c4cd5a5-4016-4668-9232-3c9cddb188a5" width="75%" />

I think stacking the labels is more readable, especially for people using magnification.  What do you think about deviating from the established layout pattern in profile for this?  We're stacking info (facility name / location) with facilities, so there would be consistency in that way.

<img src="https://images.zenhubusercontent.com/5d9def68b6de6d00018d50f2/cc47992d-e3f2-4499-b045-9556845a678a" width="75%" />

**Comment by Samara-Strauss** [comment]
> I think stacking the labels is more readable, especially for people using magnification. What do you think about deviating from the established layout pattern in profile for this? We're stacking info (facility name / location) with facilities, so there would be consistency in that way.

In terms of how it looks, it looks fine. However, I would check in @erikphansen to talk about how this would affect build for both preferred facility and this project given that the current three column layout is an established pattern. If altering the pattern in this way isn't a big deal, it's fine by me.

**Comment by andaleliz** [comment]
Awesome.  I assume it's not a big deal since it wasn't brought up when we reviewed facilites designs, but best to double check. @erikphansen, what say ye?

**Comment by erikphansen** [comment]
99% sure that'll be fine!

**Comment by Samara-Strauss** [comment]
## Research plan

- Can you please add Beverly Nelson (beverly.nelson@va.gov) as an observer? Totally optional that she's there, but she leads the VANotify team from the DEPO side of things.

I need to grab some lunch and attend a couple of meetings, but then I'll review the convo guide!

**Comment by ajakabcin** [comment]
Looks great to me @andaleliz !

**Comment by Samara-Strauss** [comment]
## Convo guide feedback

### Warm up questions

- **Tell me about any email or text notifications you've received recently** — What are we trying to learn from this question? Does this cover any notification from any site, or just VA.gov?
- **What kind of contact information do you have saved in your profile?** — I would remove this. I doubt anyone knows what they have saved. I certainly wouldn’t if someone asked me this about my bank account or health insurance or something similar.
- **How do you manage your notifications?** — I don’t think people can manage correspondence with the VA beyond opting in and out of an email or text directly, but you could probably suss out people’s needs and behaviors by following-up after they talk about correspondence with the VA and asking if they chose to received that correspondence and if they’ve ever needed to update settings around it. 
- **Let's talk about turning notifications on or off — When do you need to do this? How do you go about doing that?** — Again, I am not sure people can turn notifications on or off beyond opting in and out of texts and emails directly, so the way theis is worded is confusing to me because it implies people (1) have a place already to manage notifications (2) and do this regularly enough that they could speak to it clearly.
- **What's an instance where you might want to get more notifications from the VA?**— I don’t think you need to ask this since you already ask what kinds of notifications would be helpful.
- **What devices do you normally use to do things on your VA Profile/VA.gov?** I would remove mention of the profile because people won’t likely know what you’re talking about. Also, VA Profile is the name of the backend system and is not something veterans would be familiar with.

### Task 1

- Suggestion: You could have them go to the live VA.gov and ask this question. We might get better insight into how they’d actually navigate. Then, we could send them the prototype link for actual profile testing. But this choice is yours, so feel free to make a final call either way

## Task 2

- **What is your impression of the way the page is organized?** — I would put this question last as a follow-up and focus on the straight usability questions first.

**Comment by andaleliz** [comment]
Thanks for taking the time to review, @Samara-Strauss and @ajakabcin! Samara, I updated the guide to reflect your feedback. Below are some specific answers to your questions. I think we're in great shape for our mid-point review!

>Tell me about any email or text notifications you've received recently — What are we trying to learn from this question? Does this cover any notification from any site, or just VA.gov?

This covers notifications from any site. I'm trying to suss out what general expectations people might have around notifications, and managing them.  I reordered the questions so the question about managing notifications follows this - that question is not limited to VA notifications either.  

>Suggestion: You could have them go to the live VA.gov and ask this question. We might get better insight into how they’d actually navigate. Then, we could send them the prototype link for actual profile testing. But this choice is yours, so feel free to make a final call either way

I thought about this, but I don't want people to get distracted by the login process (we already know there's opportunity for improvement there) or have to worry about PII on the recordings.
